src/config.ts

```diff
@@ -1,8 +1,8 @@
-module.exports = {
+export const constants = {
   bearer: 'Bearer ',
   functionAppApiPath: '/api/',
   functionAppDomain: '.azurewebsites.net',
-  functionsAdminApiPath: '.azurewebsites.net/admin/functions/',
+  functionsAdminApiPath: '/admin/functions/',
   functionsApiPath: '/api/functions',
   jsonContentType: 'application/json',
   logInvocationsApiPath: '/azurejobs/api/functions/definitions/',
```

Comment:
consistency: some path end with `/` and some don't

Reply:
I think it's because some of these will be appended to others. For example `{functionAppDomain}` + `{functionsApiPath}`